tests/testthat/test-utils.R

```diff
@@ -70,3 +70,11 @@ test_that("replace_spaces works", {
     c("no_spaces_here", "none")
   )
 })
+
+test_that("pbtick() updates the progress bar", {
+  f <- function() {
+    pb <- progressr::progressor(steps = 5)
+    pbtick(pb, message = "progress!")
+  }
+  expect_condition(expect_invisible(f()))
+})
```

Comment:
I noticed our code coverage is lower for utils now. Maybe add more unit tests for when pb and/or message are `NULL`?

Reply:
That codecov report is slightly outdated (deleted it now). After [this commit](https://github.com/SchlossLab/mikropml/pull/257/commits/0b8299103add44cbe88e7dcebbac4f6099563088) the coverage for utils is 100%.